ecosystem/indexer-grpc/indexer-grpc-cache-worker/src/main.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_indexer_grpc_cache_worker::{
+    constants::APTOS_INDEXER_GRPC_CACHE_WORKER_THREAD_NAME, worker::Worker,
+};
+use tokio::runtime::Builder;
+use warp::Filter;
+
+fn main() {
+    aptos_logger::Logger::new().init();
+    aptos_crash_handler::setup_panic_handler();
+
+    let runtime = Builder::new_multi_thread()
+        .thread_name(APTOS_INDEXER_GRPC_CACHE_WORKER_THREAD_NAME)
+        .disable_lifo_slot()
+        .enable_all()
+        .build()
+        .expect("[indexer cache] failed to create runtime");
```

Comment:
aptos_runtimes? does it hurt even if this is not part of the node?

Reply:
Done.